forecast/forecaster.py

```diff
@@ -123,8 +147,12 @@ def predict(self, cluster_df, cluster, start_time, end_time):
             cluster_df.index.get_level_values(0) == cluster
         ].droplevel(0)
 
-        # Truncate cluster_df to the time range necessary to generate
-        # prediction range
+        # Truncate cluster_df to the time range necessary to generate prediction range.
+
+        # TODO(Mike): Right now, if the sequence required to predict a certain interval
+        # is not present in the data, we simply do not make any predictions (i.e. return 0)
+        # Should we produce a warning/error so the user is aware there is insufficient
```

Comment:
This sounds like a good idea, though we don't have to do it now. Good TODO comment.